frontend/public/components/resource-quota.jsx

```diff
@@ -201,6 +201,7 @@ export const QuotaGaugeCharts = ({ quota, resourceTypes, chartClassName = null }
             thresholds={gaugeChartThresholds}
             title="CPU Request"
             className={chartClassName}
+            remainderLabel="used"
```

Comment:
I'm confused. Why is "used" the `remainderLabel`?

Reply:
@spadgett oh so you think the labels are mismatched...

updating the fix